ledger/complete/wal/fadvise.go

```diff
@@ -20,6 +20,7 @@ func fadviseNoLinuxPageCache(fd uintptr, fsync bool) error {
 // If GOOS != "linux" then this function does nothing.
 // CAUTION: If fsync=true, this will call unix.Fsync which
 // can cause performance hit especially when used inside a loop.
+//nolint:unused,deadcode
```

Comment:
this code is only used by linux, macos never uses it, but the linter would complain unused or deadcode. 

skip this false negative